v2/plugins/redhat.vscode-yaml/0.3.0/meta.yaml

```diff
@@ -0,0 +1,17 @@
+publisher: ""
```

Comment:
redhat ?

Reply:
It's old notation, publisher is empty for backward compatibility